src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirTransformServiceImpl.java

```diff
@@ -377,6 +384,18 @@ public void transformPersistPatient(PatientManagementInfo patientInfo) throws Fh
         FhirOperations fhirOperations = new FhirOperations();
         org.hl7.fhir.r4.model.Patient patient = transformToFhirPatient(patientInfo.getPatientPK());
         this.addToOperations(fhirOperations, tempIdGenerator, patient);
+
+        try {
+            crClient.create().resource(patient).execute();
```

Comment:
can you talk with reagan to know exactly how this is handles in the Cleient Registry ??
Do we always create new instances ??

Reply:
Alright will do. I think I need to add updating aswell but let's confirm from @reagan-meant

Reply:
This is fine having one instance. The issue ofcourse usually is when you start the service and the fhir server is not accessible it fails